src/services/refactors/generateGetAccessorAndSetAccessor.ts

```diff
@@ -72,16 +89,14 @@ namespace ts.refactor.generateGetAccessorAndSetAccessor {
         return declaration.modifiers;
     }
 
-    function getFieldModifiers(isStatic: boolean): NodeArray<Modifier> {
+    function getFieldModifiers(isJS: boolean, isStatic: boolean, isClassLike: boolean): NodeArray<Modifier> | undefined {
+        if (isJS || !isClassLike) return undefined;

```

Comment:
you still want to handel the `static` modifier in the case of js. i would jsut make it 
```ts
return createNodeArray(
             append<Modifier>(isJS ? [] : [createToken(SyntaxKind.PrivateKeyword)],
                 isStatic ? createToken(SyntaxKind.StaticKeyword) : undefined));
```